### DISCUSSION TITLE
[2663530]pools {本体}

### DISCUSSION THREAD

**Discussion by Kalem327** [pjy612/SteamManifestCache#658] [Share]
### Steam 游戏ID

2663530

### 密钥列表

```hash
2663531;c4286bf964a6ae3ee6a54f2619f4222b3ec80a55a458d8281062b27d35afa8d0
```

### 清单附件

[2663530.zip](https://github.com/pjy612/SteamManifestCache/files/15342099/2663530.zip)

### 备注

Code:E898D3ADDD1BE44E
Emile:173962506@qq.com

**Comment by Kalem327** [comment]
是附件没有lua文件吗？我更新好了

**Reply by pjy612** [reply]
没 只是看时间以为没更，然后下载后发现更了，lua 可能仅路人需要。
